docs/deployment.md

```diff
@@ -0,0 +1,214 @@
+
+to install Forcerelay/Axon:
+```bash
+$ git clone https://github.com/synapseweb3/forcerelay
+$ cd forcerelay
+$ cargo install --path crates/relayer-cli --bin forcerelay
+```
+
+create `config.toml` in the path `~/.forcerelay/`:
+```toml
+[global]
+log_level = 'info'
+
+[mode.clients]
+enabled = true
+refresh = true
+misbehaviour = false
+
+[mode.connections]
+enabled = false
+
+[mode.channels]
+enabled = false
+
+[mode.packets]
+enabled = true
+clear_interval = 100
+clear_on_start = true
+tx_confirmation = false
+auto_register_counterparty_payee = false
+
+[rest]
+enabled = false
+host = '127.0.0.1'
+port = 3000
+
+[telemetry]
+enabled = false
+host = '127.0.0.1'
+port = 3001
+
+[[chains]]
+id = 'ckb4ibc-0'
+ckb_rpc = 'https://testnet.ckbapp.dev/'
+ckb_indexer_rpc = 'https://testnet.ckbapp.dev/'
+key_name = 'relayer_ckb_wallet'
+store_prefix = 'forcerelay'
+client_code_hash = <CLIENT_COCE_HASH>
+connection_type_args = <CONNECTION_TYPE_ARGS>
+channel_type_args = <CHANNEL_TYPE_ARGS>
+packet_type_args = <PACKET_TYPE_ARGS>
+
+[chains.packet_filter]
+policy = 'allowall'
+
+[chains.packet_filter.min_fees]
+[chains.onchain_light_clients.Axon]
+chain_id = 'axon-0'
+client_cell_type_args = <CLIENT_TYPE_ARGS>
+ibc_handler_address = <IBC_HANDLER_ADDRESS>
+
+[[chains]]
+id = 'axon-0'
+websocket_addr = "wss://rpc-alphanet-axon.ckbapp.dev/ws"
+rpc_addr = "https://rpc-alphanet-axon.ckbapp.dev"
+contract_address = <IBC_HANDLER_ADDRESS>
+restore_block_count = 10000
+key_name = "relayer_axon_wallet"
+store_prefix = "forcerelay"
+```
+
+write your keys of Axon and CKB in two different files privately, named **<your_ckb_private_file>** and **<your_axon_private_file>**, and then, import them to the installed Forcerelay/Axon:
+```bash
+$ forcerelay keys add --chain ckb4ibc-0 --secret-file <your_ckb_privkey_file>
+$ forcerelay keys add --chain axon-0 --secret-file <your_axon_privkey_file>
+```
+
+## Create IBC Connection
+to create an initial IBC connection between Axon and CKB:
+```bash
+$ forcerelay create connection --a-chain axon-0 --b-chain ckb4ibc-0
+```
+
+the connetion id of Axon, which follows the pattern `"connection-{number}"`, can be found after the command, if this is an initial run, for example, it would be `connection-0`.
+
+## Generate Port Id
+port id in Axon for transfer use is always named `transfer`, but in CKB is different, it's the lock hash of deployed `sudt-transfer` cell, which `args` can be temporarily set as ZERO:
+```toml
+# lock script of sudt-transfer
+code_hash = <SUDT_TRANSFER_TYPE_ID>
+hash_type = type
+args = 0x0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
+```
+
+calculate blake2b hash of above lock script, and let's record it as **<ST_CELL_LOCK_HASH>**.
```

Comment:
You may need some preliminary knowledge. It is recommended to use lumos to obtain the codehash of the script.
import {utils} from '@ckb-lumos/lumos'
const cell = await rpc.getLiveCell(txHash, index)
const codeHash = utils.computeScriptHash(cell.output.type)